scrc/preprocessors/extractors/judgment_pattern_extractor.py

```diff
@@ -162,94 +174,27 @@ def get_required_data(self, series: DataFrame) -> Any:
         """Do nothing"""
         
     def assign_section(self):
         sorted_lists = self.sort_dict()
-        all_judgment_markers = {
-            Language.DE: {
-                Judgment.APPROVAL: ['aufgehoben', 'aufzuheben', 'gutgeheissen', 'gutzuheissen', 'Gutheissung', 'schuldig', 'rechtmässig'],
-                Judgment.PARTIAL_APPROVAL: ['teilweise '],
-                Judgment.DISMISSAL: ['abgewiesen', 'abzuweisen', 'erstinstanzliche'],
-                Judgment.PARTIAL_DISMISSAL: ['abgewiesen, soweit',
-                                             'abzuweisen',
-                                             'abgewiesen',],
-                Judgment.INADMISSIBLE: ['Nichteintreten', 'nicht eingetreten', 'nicht einzutreten',
-                                        'wird keine Folge geleistet', 'nicht eingegangen',
-                                        'soweit darauf einzutreten ist', 'soweit auf sie einzutreten ist'],
-                Judgment.WRITE_OFF: ['abgeschrieben', 'abzuschreiben', 'gegenstandslos'],
-                Judgment.UNIFICATION: [
-                    "werden vereinigt", "werden gemeinsam beurteilt", "werden nicht vereinigt"]
-            },
-            Language.FR: {
-                Judgment.APPROVAL: ['admis', 'est annulé', 'Admet'],
-                Judgment.PARTIAL_APPROVAL: ['Admet partiellement',
-                                            'partiellement admis',
-                                            'admis dans la mesure où il est recevable',
-                                            'admis dans la mesure où ils sont recevables'
-                                            ],
-                Judgment.DISMISSAL: ['rejeté', 'Rejette', 'écarté'],
-                Judgment.PARTIAL_DISMISSAL: ['dans la mesure'],
-                Judgment.INADMISSIBLE: ['entre pas en matière', 'irrecevable', 'pas entré',
-                                        'pris en considération'],
-                Judgment.WRITE_OFF: ['retrait', 'radiée', 'sans objet', 'rayé', 'Raye'],
-                Judgment.UNIFICATION: [],
-    },
-    Language.IT: {
-        Judgment.APPROVAL: ['accolt',  # accolt o/i/a/e
-                            'annullat'],  # annullat o/i/a/e
-        Judgment.PARTIAL_APPROVAL: ['Nella misura in cui è ammissibile, il ricorso è parzialmente accolto',
-                                    'In parziale accoglimento del ricorso'],
-        Judgment.DISMISSAL: ['respint',  # respint o/i/a/e
-                             ],
-        Judgment.PARTIAL_DISMISSAL: ['Nella misura in cui è ammissibile, il ricorso è respinto',
-                                     'Nella misura in cui è ammissibile, il ricorso di diritto pubblico è respinto',
-                                     'Nella misura in cui è ammissibile, la domanda di revisione è respinta'],
-        Judgment.INADMISSIBLE: ['inammissibil',  # inamissibil o/i/a/e
-                                'irricevibil',  # irricevibil o/i/a/e
```

Comment:
Is this not necessary anymore?

Reply:
Assignment for the judgment_pattern_extractor didn't make much sense since we want to find outcome indicators which do not already exist within our set of regexes. Now we simply output the most common combinations with different n-grams as spreadsheets.